example/CountlyRNExample/Events.tsx

```diff
@@ -77,47 +37,47 @@ const startEvent = () => {
 */
 const timedEventWithSum = () => {
     // Event with sum
-    Countly.startEvent("timedEventWithSum");
-
-    const event: EventProps = {
-        eventName: "timedEventWithSum",
-        eventSum: "0.99",
-    };
+    Countly.events.startEvent("timedEventWithSum");
 
     setTimeout(() => {
-        Countly.endEvent(event);
+        Countly.events.endEvent("timedEventWithSum", undefined, undefined, 0.99);
     }, 1000);
 };
 
 const timedEventWithSegment = () => {
     // Event with segment
-    Countly.startEvent("timedEventWithSegment");
+    Countly.events.startEvent("timedEventWithSegment");
 
-    const event: EventPropsCustom_1 = {
-        eventName: "timedEventWithSegment",
-        segments: { Country: "Germany", Age: "32" },
-    };
     setTimeout(() => {
-        Countly.endEvent(event);
+        Countly.events.endEvent("timedEventWithSegment", { Country: "Germany", Age: "32" }, undefined, undefined);
     }, 1000);
 };
 
 const timedEventWithSumAndSegment = () => {
     // Event with Segment, sum and count
-    Countly.startEvent("timedEventWithSumAndSegment");
+    Countly.events.startEvent("timedEventWithSumAndSegment");
 
-    const event: EventPropsCustom_1 = {
-        eventName: "timedEventWithSumAndSegment",
-        eventCount: 1,
-        eventSum: "0.99",
-        segments: { Country: "India", Age: "21" },
-    };
     setTimeout(() => {
-        Countly.endEvent(event);
+        Countly.events.endEvent("timedEventWithSumAndSegment", { Country: "India", Age: "21" }, 1, 0.99);
     }, 1000);
 };
 // TIMED EVENTS
 
+// Test Bad Values
+const testEventWithBadValues = () => {
```

Comment:
please also test all possible valid values, int, double, long, float, string, boolean for the segmentation. Also try out giving invalid types for segmentation like put an array inside of the segmentation

Reply:
done